Backend/app/provider-platform/dynamic-offer-driver-app/Main/dynamic-offer-driver-app.cabal

```diff
@@ -287,7 +287,7 @@ library
       Storage.Queries.SlabFarePolicy
       Storage.Queries.Vehicle
       Storage.Tabular.Booking
-      Storage.Tabular.Booking.BookingLocation
+      Storage.Tabular.Booking.TripLocation
```

Comment:
why???

Reply:
It is getting removed after build by itself from cabal